src/documents/tutorial-gettingStartedWithInfusion/DefineANamespaceAndCreateAClosure.md

```diff
@@ -15,11 +15,11 @@ Infusion code generally follows a few conventions that we recommend, and that we
 
 ### Namespacing ###
 
-We define a namespace for our code: a single global variable that becomes the container for the code. This helps keep code contained, meaning there's less chance of bad interactions with other code: Anything we want to be public will be attached to this object, so all of our code will be qualified by the namespace.
+We define a namespace for our code: a single global variable that becomes the container for the code. This means there's less chance of bad interactions with other code: Anything we want to be public will be attached to this object, so all of our code will be qualified by the namespace.
 
 ### Closures ###
 
-By wrapping the rest of the code inside an anonymous function, we can separate private and public functions. Only objects or functions that are attached to the global namespace object will be publicly available. Anything else inside the anonymous function will be invisible to the rest of the world. In general we recommend against the use of private definitions since they inhibit other developers from getting value from your code. Write every function and piece of data as a public member of your namespace, with a suitable comment if you don't intend them to form a stable part of your API. Of course, make sure not to write any mutable shared state in public - in general, you should make sure any mutable state is packaged as part of a [Fluid Component](BasicComponentCreation-LittleComponents.md).
+By wrapping the code inside an anonymous function, we can separate private and public functions. Only objects or functions that are attached to the global namespace object will be publicly available. Anything else inside the anonymous function will be invisible to the rest of the world. In general, we recommend against the use of private definitions since they inhibit other developers from getting value from your code. Write every function and piece of data as a public member of your namespace, with a suitable comment if you don't intend them to form a stable part of your API. Of course, make sure not to write any mutable shared state in public - in general, you should make sure any mutable state is packaged as part of a [Fluid Component](BasicComponentCreation-LittleComponents.md).
```

Comment:
Would be good if this can link to something about model components in particular, rather than just components.